README.md

```diff
@@ -79,6 +79,10 @@ production environment:
 
     $ (.venv) ./flask-develop.sh
 
+The default generated endpoint is <http://127.0.0.1:5000/>. You should
+be able to access it, however you might receive an error message
+(the server will be empty).
```

Comment:
I think using _should_ is a bit more polite; but I will clarify this is not a valid route. I think it's good to clarify this, because many people may try to access the route they see in the logs.